service/dynamodb/dynamodbattribute/decode.go

```diff
@@ -127,6 +127,7 @@ func NewDecoder(opts ...func(*Decoder)) *Decoder {
 	d := &Decoder{
 		MarshalOptions: MarshalOptions{
 			SupportJSONTags: true,
+			SupportYAMLTags: true,
```

Comment:
Enabling `YAML` by default would be a breaking change in behavior for some applications if the struct's used by that application already include YAML tags, but their application has been (un)marshaling DynamoDB Attributes based on the struct name.